src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -795,7 +795,7 @@ static unsafe void M(dynamic d1, System.TypedReference tr)
             d1 + M,
             ( ()=>{} ) - d1, 
             d1 >> ( ()=>{} ),
-            delegate {} << d1, // TODO: This is failing due to new warning ""WRN_PrecedenceInversion"". Unclear to me what I should do here.
+            (delegate {}) << d1,
```

Comment:
Sorry, on closer inspection I was wrong about what to do here, I think the best thing would be to leave the original source alone and paste in the new actual diagnostics to VerifyDiagnostics